test/SlowTests/Issues/RavenDB_16303.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Threading;
+using Npgsql;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.ETL.SQL;
+using SlowTests.Server.Documents.ETL;
+using Tests.Infrastructure;
+using Tests.Infrastructure.ConnectionString;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_16303 : EtlTestBase
+    {
+        public RavenDB_16303(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [InlineData("{ Value: this.Guid, Type: 'Guid' }")]
+        [InlineData("this.Guid")]
+        [RequiresNpgSqlInlineData]
```

Comment:
If this change won't be Postgresql specific (see Oren's comments) then we can drop `RequiresNpgSqlInlineData`, right?

Reply:
no the connection string (line 30) and maybe the sql would have to be changed